app/models/journeys/additional_payments_for_teaching/answers_presenter.rb

```diff
@@ -156,12 +156,11 @@ def itt_academic_year
       end
 
       def text_for_subject_answer
-        policy = eligibility.policy
-        subjects = JourneySubjectEligibilityChecker.new(claim_year: Journeys.for_policy(policy).configuration.current_academic_year,
-          itt_year: journey_session.answers.itt_academic_year).current_and_future_subject_symbols(policy)
+        subjects = JourneySubjectEligibilityChecker
+          .current_and_future_subject_symbols(journey_session.answers)
 
         if subjects.many?
-          t("additional_payments.forms.eligible_itt_subject.answers.#{eligibility.eligible_itt_subject}")
+          t("additional_payments.forms.eligible_itt_subject.answers.#{journey_session.answers.eligible_itt_subject}")
```

Comment:
I'm not 100% sure we can change this. This eligibility is from the claim submission form (see the method at the top of this file), so it's created using the information from the answers. However it's a _single_ eligibility where as `current_and_future_subject_symbols` can return the subjects for multiple eligibilities. It's probably fine but might be one to check with @kenfodder to see if he can spot an issue with this change

Reply:
ok i've now reverted this back to the original implementation which now seems to work after rebasing